homeassistant/components/thermostat/zwave.py

```diff
@@ -109,6 +134,24 @@ def target_temperature(self):
     def set_temperature(self, temperature):
         """Set new target temperature."""
         # set point
+        if self._zxt_120 == 1:
```

Comment:
I guess it's fine to add it but let's extract it into a separate method. Also, you're not returning at the end of this if, should normal behavior still happen?